### PR TITLE
Wrap call-out on homepage in an "aside" element for both layouts

### PR DESCRIPTION
**Pull request by alexisreytech**

### CODE REVIEW

File: astro/src/pages/index.astro [hunk 1]
```diff
@@ -63,9 +63,11 @@ const daniela = await getEntry('profiles', 'daniela');
           often struggle to find places in their community where they can successfully shop, eat,
           and engage in community life.
         </p>
-        <p class="d-xl-none display-4 text-center text-primary py-3 mx-5 border-primary-subtle border-top border-bottom">
-          <strong>1</strong> in every <strong>4 people</strong> has a <strong>disability</strong>.
-        </p>
+        <aside>
```
Comment (brian-montgomery) [inline astro/src/pages/index.astro:66]: I would change the `p` tag to the `aside` tag in this case.

**Review by brian-montgomery — CHANGES_REQUESTED** [review]
There are two copies of the callout that switch based on screen size, you'll need to mark both of them. Line 78 is the other one.

In both cases, I would switch the `p` for the `aside` if that is what you want to do, to avoid having an empty landmark region.

**Comment by alexisreytech** [comment]
@brian-montgomery, I made a change to account for the other version of the homepage call-out. Should be good now.

**Comment by rachaelbradley** [comment]
While the call-outs could technically be asides, marking them as landmark regions adds to the number of landmark regions on the page. This adds clutter to the navigation list of landmark regions without adding much navigational benefit. I don't think we should make this change.

**Pull request closed without merging**